formulas/add-repo/README.md

```diff
@@ -0,0 +1,23 @@
+Add Repo
+========
+Exposes functions for adding a repository to a package management system.
+
+Examples
+
+    wickFormula add-repo
+
+Returns nothing.
+
+
+`addRepoApt()`
+--------------
+
+External: Adds a repository to apt.  If we find that you are running an operating system that should not be using apt we do not attempt to add it.
```

Comment:
oh.  I did not know that.  I have been adding them on other functions.  I will stop doing that.

9ba4a62a111dd03a002badde6ddca8ca77199069